drivers/hmis/lib/util/dates.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+module Util
```

Comment:
This doesn't have the driver namespace (`Hmis`) at the top level... Which means `Util` can conflict with other drivers, or the base app... So, we should go through and make sure everything else is namespaced correctly as well...

An open question to me is will GraphQL deal with it OK, or does it have to be in the root namespace?

Reply:
Ah good point, fixed. I changed modules to HmisErrors and HmisUtil.
GraphQL doesn't touch these, not sure what you mean about that.